packages/vue-components/src/TipBox.vue

```diff
@@ -386,6 +386,16 @@ export default {
     .no-border {
         border: none;
     }
+
+    @media screen and (max-width: 768px) {
+        .box-body-wrapper {
+            padding: 0.75rem;
+        }
+
```

Comment:
Could use some for the header as well to align things
```suggestion
.box-header-wrapper {
  padding: 0.75rem 0.75rem 0.1rem;
}
```

Reply:
Will do 👍